### ISSUE TITLE
Hive Elasticserach insert a date or timestamp

### ISSUE THREAD

**Issue by nmaillard** [elastic/elasticsearch-hadoop#61]
Hello everyone

I am right now playing around with this great piece of sotware.
I have looked at the code but not found a clear answer.
Say I want to push from hive into an elasticsearch index a date field how would I go about writing it?

CREATE EXTERNAL TABLE es_wrtie (
clientId String,  
mydate timestamp||string)
STORED BY 'org.elasticsearch.hadoop.hive.ESStorageHandler'
TBLPROPERTIES('es.resource' = 'test/test/')

since hive only knows timstamps would I use these, or put a strign and let elasticesearch do the trick if I format it correctly?

thnaks for your help and software

**Comment by SagarBalai** [comment]
Hi @costin 
I have seen some interesting incorrect behaviour in mapping of hive timestamp to elastic search date datatype.
Versions :: Hive -1.6, Elastic search - 2.4

In Hive, I have a column 'c1' of timestamp datatype with value as '2017-01-30 23:41:06'. In HIVE-ES integration (spark job) with third party jar 'elasticsearch-hadoop-2.4.0.jar' in spark job, 'c1' is indexed with vale '1485848466000'.
Same indexed value '1485848466000' is converted to java.util.Date using format 'yyyy-MM-dd HH:mm:ss.SSS' (same as hive timestamp) or without any format, date output is '2017-01-31 13:11:06.000'.

In short,
 '2017-01-30 23:41:06' ==> '1485848466000' ==>  '2017-01-31 13:11:06.000'
 
Here hive date and ES date is different, could you please help here.
Thanks in advance,
Sagar
